template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Startup.cs

```diff
@@ -27,6 +27,10 @@ public void ConfigureServices(IServiceCollection services)
         {
             // Add framework services.
             services.AddMvc();
+#if (OrganizationalAuth || IndividualB2CAuth)
+            services.AddJwtBearerAuthentication();
+
```

Comment:
The blank line should be before the call to `services.AddJwtBearerAuthentication();`, not after.